handlers/main.yml

```diff
@@ -2,5 +2,5 @@
 - name: restart opendkim
   service: name=opendkim state=restarted
 
-- name: restart postfix
+- name: opendkim - restart postfix
```

Comment:
I renamed this handler because handlers' names are global to a playbook (and a role can call a handler of another role) so it's better to use a kind of namespace for the handlers' names as for the variables.